index.html

```diff
@@ -260,21 +260,61 @@ <h3>
                       informando de lo que vayamos consiguiendo.
                     </p>
-                
-                  </p>
               </p>
             </div>
           </div>
         </div>
       </div>
+          <!-- whos behind -->
+          <div class="whos-behind-container" style="position: relative; z-index: 0; background: none;">
+            <div class="container">
+              <div class="row">
+                <div class="span12 whos-comming">
+                  <h3>¿Quién vendrá este año?</h3>
+                </div>
+                <div class="whos-behind-box-1">
+                  <div class="whos-behind-photo">
+                    <img src="assets/img/Carmen-Torrecillas-civio.jpg" alt="">
+                    <div class="whos-behind-social">
+                      <a href="https://twitter.com/carmen_TM_"><i class="fa fa-twitter"></i></a>
+                      <a href="https://www.linkedin.com/in/carmen-torrecillas/"><i class="fa fa-linkedin"></i></a>
+                      <a href="https://observablehq.com/@carmen-tm"><i class="fa fa-envelope"></i></a>
+                      <!--a href=""><i class="fa fa-facebook"></i></a-->
+                    </div>
+                  </div>
+                  <h4>Carmen Torrecillas</h4>
+                  <h5>Visualización de datos y frontend en Civio</h5>
+                  <p>
+                    Arquitecta por la Escuela Técnica Superior de Arquitectura de la Universidad de Granada y programadora web a través del programa de desarrollo frontend de <a href="https://adalab.es/" target="_blank">Adalab</a>. 
+                    Carmen está a cargo del desarrollo frontend, las visualizaciones de datos y el diseño gráfico de los proyectos de <a href="https://civio.es/" target="_blank">Civio</a>, una organización sin ánimo de lucro que promueve la transparencia de las instituciones y el fácil acceso a la información pública mediante la tecnología y el periodismo.                  </p>
+                </div>
+                <div class="whos-behind-box-1">
+                  <div class="whos-behind-photo">
+                    <img src="assets/img/Octopuses-review.jpg" alt="">
+                    <div class="whos-behind-social">
+                      <!--a href=""><i class="fa fa-facebook"></i></a>
+                      <a href=""><i class="fa fa-twitter"></i></a>
+                      <a href=""><i class="fa fa-linkedin"></i></a>
+                      <a href=""><i class="fa fa-envelope"></i></a-->
+                    </div>
+                  </div>
+                  <h4>Tú</h4>
+                  <h5>Geoinquieto</h5>
+                  <p>
+                    Nadie mejor que tú mismo para saber quién eres. Vendrás a la geocamp, hablarás de GIS y disfrutarás de un día en al preciosa Isla de San Simón.
```

Comment:
```suggestion
                    Nadie mejor que tú mismo para saber quién eres. Vendrás a la Geocamp, hablarás de GIS y disfrutarás de un día en al preciosa Illa de San Simón.
```

Na dos chorradas, lo de la mayúscula y lo de usar `Illa` como en el resto de la portada.

Reply:
en al preciosa llla -> en la preciosa Illa